lib/Sabberworm/CSS/Parser.php

```diff
@@ -434,6 +436,8 @@ private function parsePrimitiveValue() {
 			$oValue = $this->parseColorValue();
 		} else if ($this->comes('url', true)) {
 			$oValue = $this->parseURLValue();
+		} else if ($this->comes('calc', true) || $this->comes('-webkit-calc', true)) {
```

Comment:
Support `-moz-calc` as well. Also, is there an `-o-calc`?

Reply:
Done for `-moz-calc`, but `-o-calc` does not seem to exist https://www.w3schools.com/cssref/func_calc.asp